packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ConfigurationFiles/index.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import {
+  Element,
+  Collapsible,
+  Text,
+  Button,
+  Stack,
+  Grid as BaseGrid,
+} from '@codesandbox/components/lib/';
+import getDefinition from '@codesandbox/common/lib/templates';
+import { resolveModule } from '@codesandbox/common/lib/sandbox/modules';
+
+import { useOvermind } from 'app/overmind';
+import styled, { withTheme } from 'styled-components';
+import { TypescriptIcon } from 'app/components/TypescriptIcon';
+import {
+  NetlifyIcon,
+  PrettierIcon,
+  NPMIcon,
+  ZeitIcon,
+  JSIcon,
+  CodeSandboxIcon,
+} from './Icons';
+
+const getIcon = name => {
+  const icons = {
+    'netlify.toml': NetlifyIcon,
+    'package.json': NPMIcon,
+    'now.json': ZeitIcon,
+    '.prettierrc': PrettierIcon,
+    'jsconfig.json': JSIcon,
+    'tsconfig.json': TypescriptIcon,
+    'sandbox.config.json': CodeSandboxIcon,
+  };
+
+  return icons[name] || JSIcon;
+};
+
+const Grid = styled(BaseGrid)`
+  grid-template-columns: 1fr 100px;
+  grid-gap: ${({ theme }) => theme.space[4]}px;
+`;
+
+export const ConfigurationFilesComponent = ({ theme }) => {
+  const {
+    state: {
+      editor: { currentSandbox },
+    },
+    actions: { files, editor },
+  } = useOvermind();
+
+  const { configurationFiles } = getDefinition(currentSandbox.template);
+
+  const createdPaths = {};
+  const restPaths = {};
+
+  Object.keys(configurationFiles)
+    .sort()
+    .forEach(p => {
+      const config = configurationFiles[p];
+
+      try {
+        const module = resolveModule(
+          p,
+          currentSandbox.modules,
+          currentSandbox.directories
+        );
+        createdPaths[p] = {
+          config,
+          module,
+        };
+      } catch (e) {
+        restPaths[p] = {
+          config,
+        };
+      }
+    });
+
+  return (
+    <>
+      <Collapsible title="Configuration Files" defaultOpen>
+        <Stack
+          direction="vertical"
+          gap={6}
+          style={{ padding: `0 ${theme.space[3]}px` }}
```

Comment:
Should padding be treated like margin in Element?

`Stack paddingX={3}>` instead of this ^

Reply:
That was the first thing I tried but it does:

![Screenshot 2020-01-16 at 13 58 11](https://user-images.githubusercontent.com/1051509/72527179-7641cf00-3868-11ea-8883-4e9910dcf5b1.png)


This may have something to do with the css prop in styled-components